.github/workflows/First.yml

```diff
@@ -6,11 +6,12 @@ on:
 jobs:
   build_n_test:
     runs-on: ubuntu-latest
+    container: mesmatyas/mosze_main:latest
     steps:
       - uses: actions/checkout@v2
-      - name: Install valgrind and cppcheck
-        run: sudo apt-get install -y valgrind cppcheck
-      - name: run 
+      - name: change permissions
+        run:  chmod +x test/generate_outputs.sh
```

Comment:
nem latom, hogy az install helyere ez hogy jon be. Eddig nem kellett?

Reply:
Csak megszokásból maradt bent.